config.js

```diff
@@ -9,20 +9,22 @@ function getChalkColors(defaultConfig, overrideConfig) {
 
 function buildChalkFunction(config) {
 	var chalkColors = {};
-	for (var color in config.colors) {
-		if (config.colors.hasOwnProperty(color)) {
-			var colorFunc = chalk;
-			//build up the chalk function
-			try {
-				config.colors[color].forEach(function(style) {
-					colorFunc = colorFunc[style];
-				});
-				if (typeof colorFunc === 'undefined') {
-					throw new Error('Unsupported colorFunc');
+	if (config) {
+		for (var color in config.logs) {

```

Comment:
can likely move the entire for loop into a function so that lines and logs can simply call the function to build up their colors.